CHANGELOG_iOS.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## [1.66.123](https://github.com/brave/brave-browser/releases/tag/v1.66.123)
+
+ - Fixed a crash when dismissing alerts/share sheet menu in certain cases. ([#38651](https://github.com/brave/brave-browser/issues/38651))
+ - Fixed "Add to Calendar" prompt not displaying when invite URLs are loaded on NTP and other websites. ([#38548](https://github.com/brave/brave-browser/issues/38548))
+ - Fixed download prompts appearing when switching tabs. ([#38551](https://github.com/brave/brave-browser/issues/38551))
```

Comment:
nit: lets add `in certain cases` as I don't think it happened every single time.